tools/deploy.py

```diff
@@ -80,7 +80,8 @@
   if package_key == "bose" and BOSE_APPKEY:
     # write the app key into a app_key.lua (overwrite if exists already)
     subprocess.run(["touch -a ./src/app_key.lua && echo \'return \"" + BOSE_APPKEY +  "\"\n\' > ./src/app_key.lua"], cwd=driver, shell=True, capture_output=True)
-  subprocess.run(["zip -r ../edge.zip config.yml fingerprints.yml $(find profiles -name \"*.y*ml\") $(find . -name \"*.lua\") -x \"*test*\""], cwd=driver, shell=True,  capture_output=True)
+  subprocess.run(["zip -r ../edge.zip config.yml fingerprints.yml $(find profiles -name \"*.y*ml\") $(find . -name \"*.lua\") -x \"*test*\""], cwd=driver, shell=True, capture_output=True)
```

Comment:
Do you have magical white-space detecting glasses or did your Python formatter find this for you?

Reply:
I just saw it.